packages/design-system/src/components/InlineError/InlineError.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { InlineError } from './InlineError';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof InlineError> = {
+  title: 'Components/InlineError',
+  component: InlineError as any,
+  args: {
+    children: 'A date of birth cannot be in the future.',
+  },
+  parameters: {
+    docs: {
+      underlyingHtmlElements: ['span'],
+    },
+  },
+};
+export default meta;
+
+type Story = StoryObj<typeof InlineError>;
+
+export const Default: Story = {};
```

Comment:
id prop didn't update markup in storybook

Reply:
This seems to be a problem with Storybook. It only works if I either refresh the page or get all hacky and add a `key` prop like this to force it to always re-render when an arg changes:
```ts
export const Default: Story = {
  render: (args) => <InlineError {...args} key={JSON.stringify(args)}>{args.children}</InlineError>
};
```

Reply:
It appears to be an issue in the React version of our Storybook too

Reply:
I'm going to leave it broken for now because it's not really related to this change.